ClassTranscribeDatabase/CommonUtils.cs

```diff
@@ -166,5 +158,38 @@ public static string GetMediaName(Media media)
 
             return name;
         }
+
+        public static CourseOffering GetRelatedCourseOffering(Entity entity)
+        {
+            switch (entity)
+            {
+                case Course c:
+                    return c.CourseOfferings?
+                        .Where(co => co.IsDeletedStatus == Status.Active && !string.IsNullOrEmpty(co.FilePath))
```

Comment:
Delete's are soft, so I don't know if the above check is useful.
By the way, it may already be covered by the following filter-
https://github.com/classtranscribe/WebAPI/blob/0cc8cde7d8b735262b9042147598af6c67391ba3/ClassTranscribeDatabase/CTDbContext.cs#L152

Reply:
Ah yes, it's already covered with the query filters. Thank you!